### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShalokShalom**
In addition to this little correction do I consider to rephrase the following sentence as well, since I feel a little bit confused by it:

> 
All of the code produced by Elchemy can be easily read and analyzed without taking a single look at the source code

### CODE REVIEW

**Comment by wende** [comment]
What would you feel like rephrasing it to?
As I'm not native my English is obviously far from perfect, so I'm open to any suggestions regarding syntactical aspects

**Comment by ShalokShalom** [comment]
Which source code you mean?

> without taking a single look at the source code

**Comment by wende** [comment]
@ShalokShalom 
Ok. I'll try to rephrase it.
What I mean is there's plenty of languages that output to other languages.
And even if you can understand JavaScript, that doesn't mean an output of Scala.js (Scala to JavaScript) can be reasoned about without looking at the Scala code that produced it.
It's just completely unreadable. What does it mean to the developer? 
If someone wrote a part of the system in Scala.js unless you know Scala you won't understand what the code does.

There are however languages that produce outputs that are **meant to** be read. Like for example CoffeeScript. You don't have to know CoffeeScript to read and understand it's JavaScript output.
You can just look at the JS output and you'll be fine.

The same is true for Elchemy (Well, it got a little bit worse lately, but still readability is our primary concern). You don't need to understand Elchemy. You know only Elixir? That's fine. Just open `outputfile.elchemy.ex` and you should be fine. 

That gives a very important boost to the learning curve for people coming from Elixir (who obviously are our main target after all)

**Comment by ShalokShalom** [comment]
"All of the code produced by Elchemy can be easily read and analyzed without taking a single look at the original source."

You could also use the word idiomatic?

Like: The produced Elixir code is idiomatic, performant and it's quite hard to detect the difference between hand written code and such one that is produced by Elchemy.

**Comment by wende** [comment]
Not being able to tell a difference is a little bit of a stretch. Right now it's quite obvious if it was produced by Elchemy. 
And although there are outputs like that:

screen shot 2018-02-04 at 23 21 36

Most of the time they actually look like this

screen shot 2018-02-04 at 23 23 03

Which although still easy to read and reason about, it's quite obvious it wasn't hand-written. Mostly because of the awkward parenthesis use.

I like the use of the word idiomatic here.
I think the first part of the second sentence and the other half of the first one add up to both easy to read and accurate definition.

**Comment by ShalokShalom** [comment]
Yes, perfectly. Will you do it?

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Elchemy lets you write simple, fast and quality type safe code while leveraging
 - **Easy and type-safe interop**: You can call Elixir/Erlang without any extra boiler-plate. All the calls you make are checked in terms of type-safety as thoroughly as possible
 - **All the best of Elm and Elixir**: Elchemy inherits what's best in Elm - type safety, inference and extreme expressiveness, but also what's best in Elixir - Doc-tests, tooling and obviously the entire BEAM platform.
 - **Nearly No runtime errors** - Elchemy's type system **eliminates almost all runtime errors**. With a shrinking set of edge cases, your entire app will be as safe as the parts written in Elixir are.
-- **Beatiful and fully readable output** - All of the code produced by Elchemy can be easily read and analyzed without taking a single look at the source code
+- **Beautiful and fully readable output** - The produced code is idiomatic, performant and can be easily read and analyzed without taking a single look at the original source.
```
Comment (wende) [inline README.md]: @ShalokShalom opinions?

Comment (ShalokShalom) [reply README.md]: Fine. The other PR is still open FYI

Comment (wende) [reply README.md]: Yup. Preparing a longer answer to that right now. It seems to touch much larger issue than just English semantics 😄
